src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -264,5 +264,39 @@ public static void SafeX509ChainHandle_InvalidHandle_StaticObject()
 
             Assert.Same(firstCall, secondCall);
         }
+
+        [Fact]
+        [OuterLoop( /* May require using the network, to download CRLs and intermediates */)]
+        public static void VerifyWithRevocation()
+        {
+            using (var cert = new X509Certificate2(Path.Combine("TestData", "MS.cer")))
+            using (var chain = new X509Chain())
+            {
+                chain.ChainPolicy.VerificationFlags =
+                    X509VerificationFlags.AllowUnknownCertificateAuthority;
+
+                chain.ChainPolicy.VerificationTime = cert.NotBefore.AddHours(2);
+                chain.ChainPolicy.RevocationMode = X509RevocationMode.Online;
+                chain.ChainPolicy.RevocationFlag = X509RevocationFlag.EntireChain;
+
+                bool valid = chain.Build(cert);
+                Assert.True(valid, "Chain Built Validly");
+
+                // Since the network was enabled, we should get the whole chain.
+                Assert.Equal(3, chain.ChainElements.Count);
+
+                Assert.Equal(0, chain.ChainElements[0].ChainElementStatus.Length);
+                Assert.Equal(0, chain.ChainElements[1].ChainElementStatus.Length);
+
+                X509ChainStatus[] rootElementStatus = chain.ChainElements[2].ChainElementStatus;
+                Assert.InRange(rootElementStatus.Length, 0, 1);
```

Comment:
The root is trusted on my Windows machine at work.  Either because it's a Windows thing, or a corpnet thing.  But it isn't trusted by my Linux machine.